packages/tools/kadena-cli/src/devnet/utils/simulation/file.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+import path from 'path';
+import { services } from '../../../services/index.js';
+
+export interface IFileData {
+  timestamp: number;
+  from: string;
+  to: string;
+  amount: number;
+  requestKey: string;
+  action: TransferType | TokenActionType;
+}
+
+export interface IFileError {
+  error: string;
+}
+
+export type TransferType =
+  | 'fund'
+  | 'transfer'
+  | 'cross-chain-transfer'
+  | 'safe-transfer'
+  | undefined;
+
+export type TokenActionType = 'create' | 'mint' | 'transfer' | undefined;
+
+export function createDir(directory: string): void {
+  if (!fs.existsSync(directory)) {
+    fs.mkdirSync(directory);
+  }
+}
+
+export function createFile(folder: string, filename: string): string {
+  const exampleData: IFileData = {
+    timestamp: 0,
+    from: '',
+    to: '',
+    amount: 0,
+    requestKey: '',
+    action: undefined,
+  };
+  const directory = path.join(`${folder}/`);
+  createDir(directory);
+  const filepath = path.join(directory, filename);
+  const headers = `${Object.keys(exampleData).join(',')}\n`;
+  fs.writeFileSync(filepath, headers);
```

Comment:
please use the `service.filesystem` instead

Reply:
Fixed